converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Modules/DicomModulesExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Client.Extensions;
+using Microsoft.Health.Dicom.Client;
+using Microsoft.Health.DicomCast.Core.Configurations;
+using Microsoft.Health.DicomCast.Core.Features.DicomWeb.Service;
+using Microsoft.Health.Extensions.DependencyInjection;
+using Polly;
+using Polly.Extensions.Http;
+using Microsoft.Health.Client.Authentication;
+
+namespace Microsoft.Health.DicomCast.Core.Modules;
```

Comment:
Oh yeah, I forgot to mention this - 

I talked to Will and you guys were correct. Namespaces for Dependency Injection Extension classes should be "Microsoft.Extensions.DependencyInjection". It makes the available extension for DI more easily discoverable when someone consumes our packages. So lets rename this namespace